code/modules/mob/living/carbon/xenomorph/xeno_defines.dm

```diff
@@ -317,7 +317,8 @@
 	var/upgrade = XENO_UPGRADE_INVALID
 	///sunder affects armour values and does a % removal before dmg is applied. 50 sunder == 50% effective armour values
 	var/sunder = 0
-	var/fire_resist_modifier = 0
+	///temporary modifiers to armor
+	var/list/resist_modifiers = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, BIO = 0, FIRE = 0, ACID = 0)
```

Comment:
Actually, what about repurposing hard armor for xenos specifically? They don't use it at all right now, far as I know, and you could just change their get_hard_armor/modify_by_armor.

Reply:
@TiviPlus The reason I don't modify armour directly is because then it is impacted by sunder. Jelly not providing fire immunity would be kinda cringe.

@QualityVan Part of refactoring fire stuff was to specifically get away from the weird hard armour interactions. Armour should behave consistantly IMO. Although I do have some plans for hard armour in general, that's part of a way bigger piece.